app/renderer/js/tray.ts

```diff
@@ -1,17 +1,15 @@
 'use strict';
-const path = require('path');
-
-const electron = require('electron');
-
-const { ipcRenderer, remote } = electron;
+import { ipcRenderer, remote, WebviewTag, NativeImage } from 'electron';
```

Comment:
Can somebody explain `WebviewTag`, `NativeImage`?

Reply:
@priyank-p Same here, as in the OpenDialogOptions. Even though, this is also a right way to do this. I think we'll have to prefer ```Electron.type``` method to avoid unnecesary imports.